lib/anagram.rb

```diff
@@ -8,17 +8,26 @@ def finder(word_two)
     word_1 = @word
     word_2 = word_two
 
-    first_word = word_1.downcase.chars.sort.join("")
-    second_word = word_2.downcase.chars.sort.join("")
+    first_word = word_1.downcase.chars.sort #.join("")
+    second_word = word_2.downcase.chars.sort #.join("")
 
     if first_word == second_word
       "It's an anagram"
-    elsif first_word == /[aeiou]/ || second_word == /[aeiou]/
+    elsif first_word.match?(/[aeiou]/) && second_word.match?(/[aeiou]/)
```

Comment:
Are you checking if both words have vowels or just any of the inputed words. 
- You're using  ```&&``` so this ```elsif``` will only return true if both statements are true. Maybe or ```||``` would be good here
- You're returning "You'll need a vowel." So you might want to check if your if isn't true so you can use ```!``` or ```not``` so something like ```!first_word.match?(/[aeiou]/) && !second_word.match?(/[aeiou]/)```

Reply:
checking if the inputted word has a vower otherwise it's not considered a word.